StoreKitUnitTests/StoreProductTests.swift

```diff
@@ -131,7 +135,11 @@ class StoreProductTests: StoreKitConfigTestCase {
 
         expect(storeProductSet).to(haveCount(1))
 
-        let storeProduct = try XCTUnwrap(storeProductSet.first)
+        let sk2Product = try XCTUnwrap(storeProductSet.first)
+        let storeProduct = StoreProduct.from(product: sk2Product)
+
+        // Can't use `===` because `SK2Product` is a `struct`
```

Comment:
I opened https://github.com/Quick/Nimble/pull/955 so that `Nimble` doesn't even allow that.